cli/connhelper/ssh/ssh.go

```diff
@@ -4,10 +4,15 @@ package ssh
 
 import (
 	"net/url"
+	"os"
 
 	"github.com/pkg/errors"
 )
 
+var (
+	dockerSSHOptions = os.Getenv("DOCKER_SSH_OPTIONS")
+)
```

Comment:
DOCKER_HOST_SSH_OPTIONS, because it has nothibg to do with docker build --ssh

Reply:
Changed var name & split string.